editor/code_editor.cpp

```diff
@@ -2087,13 +2044,42 @@ CodeTextEditor::CodeTextEditor() {
 	warning_button->set_tooltip_text(TTR("Warnings"));
 	set_warning_count(0);
 
+	status_bar->add_child(memnew(VSeparator));
+
+	// Zoom
+	zoom_button = memnew(MenuButton);
+	status_bar->add_child(zoom_button);
+	zoom_button->set_flat(true);
+	zoom_button->set_v_size_flags(SIZE_EXPAND | SIZE_SHRINK_CENTER);
+	zoom_button->set_text("100 %");
+	zoom_button->set_tooltip_text(TTR("Zoom factor"));
+	zoom_button->get_popup()->add_item("25 %");
```

Comment:
Should this space be here? (the one between number and `%`, it looks odd).

Reply:
I'm not sure but it seems the editor has a space, too. (Just not on the tooltip). 
![image](https://github.com/godotengine/godot/assets/66727710/471fc029-3ebb-4a14-9e6f-24b7955fcace)
If yes, keeping it would be consistent.

Reply:
Ah yeah that was also another thing I checked against for consistency. I thought it was odd at first but after trying it without the space, the menu looked like a contiguous blob of various sized circles due to all the rows of 00%. The default editor font also makes this worse because the percent sign is very big and prominent.